deepinv/physics/functional/radon.py

```diff
@@ -113,9 +159,29 @@ def __init__(
         if theta is None:
             self.theta = torch.arange(180)
         self.dtype = dtype
+        self.parallel_computation = parallel_computation
+        self.fan_beam = fan_beam
+        self.fan_parameters = fan_parameters
+        if fan_beam:
+            if self.fan_parameters is None:
+                self.fan_parameters = {}
+            if not "pixel_spacing" in self.fan_parameters.keys():
+                self.fan_parameters["pixel_spacing"] = 0.077
+            if not "source_radius" in self.fan_parameters.keys():
+                self.fan_parameters["source_radius"] = 57.5
```

Comment:
should this be somehow relative to the number of pixels in the image?

Reply:
You are right that the values should adapt to the image size. I changed the default value of the pixel spacing to 1/im_with.